daemon/cmld.c

```diff
@@ -585,28 +587,26 @@ cmld_container_config_sync_cb(container_t *container, container_callback_t *cb,
 	IF_FALSE_RETURN_ERROR(container_get_state(container) == COMPARTMENT_STATE_REBOOTING ||
```

Comment:
we should discuss whether this should be a DEBUG log message

Reply:
yes we should. is done now